bluetooth_mesh/application.py

```diff
@@ -916,7 +894,7 @@ def dev_key_message_received(
                 return
 
     def update_model_configuration(
-        self, model_id: int, configuration: Mapping[str, Any]
+        self, model_id: int, vendor_id: int, configuration: Mapping[str, Any]
```

Comment:
Consistency: make `model_id` a `Tuple[Optional[int], int]` (see models/base.py:61) and pass `vendor_id` as the *first* element of that tuple.